stdlib/private/DifferentiationUnittest/DifferentiationUnittest.swift

```diff
@@ -257,18 +257,27 @@ extension Tracked where T : Differentiable & FloatingPoint, T == T.TangentVector
   }
 }
 
-// Differential operators for `Tracked<Float>`.
-public func gradient(
-  at x: Tracked<Float>, in f: @differentiable (Tracked<Float>) -> Tracked<Float>
-) -> Tracked<Float> {
-  return pullback(at: x, in: f)(1)
+// Differential operators for `Tracked<T>`.
+
+public func gradient<T, U>(
```

Comment:
Single gradient operator definitions for `(...) -> Tracked<T>` functions is great!
For reference: [TF-927](https://bugs.swift.org/browse/TF-927) tracks the confusing type-checker error we encountered this morning.